marklogic-data-hub-central/ui/src/components/raw-data/raw-data-card-view/raw-data-card-view.tsx

```diff
@@ -67,6 +65,44 @@ const RawDataCardView = (props) => {
     }
   }
 
+  const displayDocumentSources = (item) => {
+    let sources = item.hubMetadata?.sources.map((el) => {
+      return el.name;
+    }).join(", ");
```

Comment:
@timur-isangulov  @rahulvudutala  @yasht2010  Should consider that a source name could have a comma in it. We don't expect that, but it's also not prohibited. That's a reason why I favor some sort of structured grouping any time we list sources, as opposed to relying on textual delimiters.

Reply:
@rjrudin This is just an overview of source names limited to 23 characters. The accurate Record metadata can be seen in the Detail view. I assume it should be alright for one edge case. 
@yasht2010 can confirm.